etc/tendrl/email.conf.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Make this also as sample and while copying in rpm install time it should be named properly.